extensions/amp-apester-media/0.1/validator-amp-apester-media.protoascii

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-apester-media
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-apester-media extension .js script"
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-apester-media"
+    dispatch_key: true
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-apester-media-(latest|0\\.1)\\.js"
+  }
+  attrs: {
+    name: "type"
+    value: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-apester-media.html"
+}
+tags: {  # <amp-apester-media>
+  html_format: AMP
+  tag_name: "AMP-APESTER-MEDIA"
+  also_requires_tag: "amp-apester-media extension .js script"
+  attrs: {
+    name: "data-apester-media"
+    mandatory: false
```

Comment:
I'm more in preference to merge this PR sooner so that it can be tried out even if validator is still in progress. @powdercloud would you recommend validator rules to be moved to a separate PR or are we close enough already?